Rules/Rules.csproj

```diff
@@ -53,6 +45,17 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net451'">
+    <PackageReference Include="System.Management.Automation" Version="6.0.0-alpha13" />
+  </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Management.Automation" Version="6.0.2" />
+    <PackageReference Include="Microsoft.Management.Infrastructure" Version="1.0.0-alpha08" />
```

Comment:
could this just be `Version="1.0.0-alpha*` (that's what it is in PowerShell repo)

Reply:
Yes, I think it is reasonable to always take the latest version of a pre-release package.